### ISSUE TITLE
`bootstrap` / `database install` calls `database migrate` without a way to run `schema apply` in the middle

### ISSUE THREAD

**Issue by viters** [directus/directus#9911]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

There is no way to run `database install` without  `database migrate` being automatically called afterwards. It is a crucial problem, while trying to implement a migration job:
1. Install initial database
2. Apply schema changes
3. Apply migrations

Consider situation, where you develop a new feature, you create a new collection and you need to create a migration that creates initial data for this collection. To achieve this, schema changes must run before migrations. So it is easy:
```
npx directus schema apply
npx directus database migrations latest
```
But that script will fail, when you deploy a new environment, because schema apply contains code:
```typescript
if ((await isInstalled()) === false) {
	logger.error(`Directus isn't installed on this database. Please run "directus bootstrap" first.`);
	database.destroy();
	process.exit(0);
}
```
This is invalid - I cannot run `directus bootstrap`, because it will run migrations that will fail without applying schema. In other words, I need to apply:
```
npx directus database install **--skipMigrations**
npx directus schema apply
npx directus database migrations latest
```

### To Reproduce

N/A

### What version of Directus are you using?

v9.0.1

### What version of Node.js are you using?

N/A

### What database are you using?

N/A

### What browser are you using?

N/A

### What operating system are you using?

N/A

### How are you deploying Directus?

N/A

**Comment by rijkvanzanten** [comment]
Agreed. `database install` should _not_ run the migrations. We should remove this line:

https://github.com/directus/directus/blob/e6d72bc0ec3fa904cd232628104d9c3f02ec4358/api/src/cli/commands/database/install.ts#L11

**Comment by viters** [comment]
@rijkvanzanten 

Thanks for review. I'll send PR today.

**Comment by eduardoPinto12** [comment]
I have a similar case, where I want to run on a docker image the command `database install` -> schema -> migrations, but unlike `directus bootstrap`, it will fail if the db is already installed and not run the schema apply and migrations, how can I ignore install if db is already installed?

**Comment by rijkvanzanten** [comment]
Mind opening that as a new Q&A Discussion? That way we don't spam the notifications of the others on this PR thread 🙂
